subiquity/common/types.py

```diff
@@ -325,6 +340,8 @@ class GuidedCapability(enum.Enum):
     DIRECT = enum.auto()
     LVM = enum.auto()
     LVM_LUKS = enum.auto()
+    ZFS = enum.auto()
```

Comment:
I presume it's the right call that ZFS.supports_manual_customization() is False? It makes sense but I just wanted to check given that that method appeared after you started working on this branch.

Reply:
We could enable it but it's sort of a weird semi-supported state.  I think it would make more sense when we have zfs manual partitioning support.  For now all you'd be able to do would be to configure other partitions on secondary disks.

I lean toward not including ZFS in supports_manual_customization until we have manual zfs config support.

Reply:
yes that's fair. it's not like the desktop client cares yet :-)